### ISSUE TITLE
Feature Request: Increase viewable items in style configurator from 30 to 50

### ISSUE THREAD

**Issue by kyzer-davis** [notepad-plus-plus/notepad-plus-plus#10206]
### Description of the Issue
Style Configurator for Global Style only only shows 30 items from stylers.xml
Found while working on feature request which requires ~~40~~ 50 items in the Global Style XML.

### Steps to Reproduce the Issue
1. Open stylers.xml and add XML entry to `<GlobalStyles>`
2. Close/Reopen or Build N++
3. Settings > Style Configurator... > Global Styles

### Expected Behavior
31st item from stylers.xml is shown in the scroll menu

### Actual Behavior
31st item is clipped.

### Screenshot
I have added Line 1396 to stylers.xml for my WIP feature and reloaded. After, in style configurator, line 1404  for "URL Hovered" is not present in the second columns list of entries. The scrollbar stops at 30th item "Inactive tabs" in that XML segment .
![image](https://user-images.githubusercontent.com/867657/126368732-66373994-174d-4343-84dc-a0111abb5bd0.png)

### Analysis
This file/line appears to be the source of the problem:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/scintilla/include/SciLexer.h#L232

Values 31 through 38 acceptable from my testing. See below with change to:
`#define SCE_STYLE_ARRAY_SIZE                38`
![image](https://user-images.githubusercontent.com/867657/126375663-78c2ff95-23a8-41a5-8f6c-546e5bcea271.png)

However, I found that increasing the value to 39 or higher causes N++ to crash upon open. (Need at least 40 for my WIP Feature)
`#define SCE_STYLE_ARRAY_SIZE                39`

Unsure what else would be required to go from 30 to ~~40~~ 50 without a crash.

### Debug Information
Fresh `Unicode Debug | x64` Build with only https://github.com/kyzer-davis/notepad-plus-plus/commit/99387bf58785fa78eb47b884fc9e50ed1560b117 to test this:
```
Notepad++ v8.1.1   (64-bit)
Build time : Jul 20 2021 - 13:17:55
Path : C:\Users\username\Desktop\notepad-plus-plus\PowerEditor\visual.net\x64\Unicode Debug\Notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 2009
OS Build : 19042.1052
Current ANSI codepage : 1252
Plugins : none
```

Edit: 40 -> 50

**Comment by donho** [comment]
@kyzer-davis 
According my test, it allows to increase  SCE_STYLE_ARRAY_SIZE upto 33 for not having crash.
BTW, why do you need to increase this value?

**Comment by donho** [comment]
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/e9cf428388fc4e7f927b269be081d5876abb9963

**Comment by donho** [comment]
This issue is solved due to removal of e9cf428
